index.html

```diff
@@ -180,7 +180,28 @@ <h4 class="setup-similar-title">Похожие персонажи</h4>
     <p class="setup-similar-label">Петр Петров</p>
   </div>
 </template>
-
+<script>
+var fireballSize = 22;
+var wizardSpeed = 3;
+var wizardWidth = 70;
+function getWizardHeight() {
+  return 1.337 * wizardWidth;
+}
+function getWizardX(gameFieldWidth) {
+  return (gameFieldWidth - wizardWidth)/2;
```

Comment:
Необходимо добавить проблемы между оператором и операндами.
```suggestion
  return (gameFieldWidth - wizardWidth) / 2;
```